### PR TITLE
Implement real encoder distance math

### PR DESCRIPTION
**Pull request by IshraqBhuiyan**
Adjusts the encoder distances we receive from the encoder using the `getDistance()` method to give us a real distance based upon empirical data, as shown in the graph and data table below from testing. The constants `REAL_DISTANCE_CONVERSION_FACTOR` and `REAL_DISTANCE_OFFSET` can be changed as per further empirical data, to gain further accuracy for real distance.
_NOTE:_ There was an error with the calculation of the `DRIVETRAIN_ENCODER_DISTANCE_PER_PULSE` in the **testing-yolo** branch, which was used to acquire the empirical data on which the constants are currently set at, so before actually implementing this, these constants must be changed to match the current encoder properties and constants. 
tl;dr: **testing-yolo** branch messed up the data
image

### CODE REVIEW

**Comment by IshraqBhuiyan** [comment]
Justin's absolute value distance method 'getDistanceAbsolute()' was implemented and replaced 'getDistance()'.

**Comment by IshraqBhuiyan** [comment]
Nicole's/Kevin's/Justin's/Obama's absolute value distance method `getDistanceAbsolute()` was implemented and replaced `getDistance()`.

**Comment by yuna9** [comment]
Obsoleted by the fact that our encoders are pretty accurate!
